client/extensions/zoninator/state/locks/reducer.js

```diff
@@ -0,0 +1,37 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { combineReducers, createReducer, keyedReducer } from 'state/utils';
+import {
+	ZONINATOR_REQUEST_LOCK_ERROR,
+	ZONINATOR_RESET_LOCK,
+	ZONINATOR_UPDATE_LOCK,
+} from '../action-types';
+
+export const blocked = createReducer( false, {
+	[ ZONINATOR_UPDATE_LOCK ]: () => false,
+	[ ZONINATOR_REQUEST_LOCK_ERROR ]: () => true,
+} );
+
+export const created = createReducer( 0, {
+	[ ZONINATOR_RESET_LOCK ]: ( state, { time } ) => time,
```

Comment:
Is there a reason to use `{ time }` here, and `action` below, or can we make them consistent?

Reply:
I did it so I didn't have to re-use `created` variable name, which is already used as the reducer's name and this prohibited by our style guide.

Reply:
I think the destructured one is actually quite elegant; in order to avoid variable name shadowing for the others, you could've aliased them, e.g.

```js
export const expires = createReducer( 0, {
	[ ZONINATOR_UPDATE_LOCK ]: ( state, { expires: expiresValue } ) => expiresValue,
} );
```